### PR TITLE
Create admins page on admin

### PR DESCRIPTION
**Pull request by romanortynskyi**

### CODE REVIEW

File: src/components/enhanced-table/EnhancedTable.jsx [hunk 1]
```diff
@@ -33,13 +33,20 @@ const EnhancedTable = ({ fetchService, externalFilter }) => {
   const { loading, fetchData } = useAxios({ service: fetchService, fetchOnMount: false })
 
   const getData = useCallback(async () => {
+    let status = null
+console.log(filters)
```
Comment (Yurenko) [inline src/components/enhanced-table/EnhancedTable.jsx:37]: Delete console.log

Comment (romanortynskyi) [reply src/components/enhanced-table/EnhancedTable.jsx]: done

File: src/constants/translations/en/admin-table.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "adminsTab": "Admins Tab",
+  "name": "Name",
+  "email": "Email",
+  "lastLogin": "Last login",
+  "signedUp": "Signed up",
+  "all": "All",
+  "status": "Status",
+  "active": "Active",
+  "blocked": "Blocked"
```
Comment (Yurenko) [inline src/constants/translations/en/admin-table.json:10]: Create the same for Ukrainian language.

Comment (romanortynskyi) [reply src/constants/translations/en/admin-table.json:10]: Admin portal won't have Ukrainian translations according to our design.

File: src/components/enhanced-table/EnhancedTable.jsx [hunk 3]
```diff
@@ -33,13 +33,20 @@ const EnhancedTable = ({ fetchService, externalFilter }) => {
   const { loading, fetchData } = useAxios({ service: fetchService, fetchOnMount: false })
 
   const getData = useCallback(async () => {
+    let status = null
```
Comment (Tolik170) [inline src/components/enhanced-table/EnhancedTable.jsx:36]: can this will work?
const status  = externalFilter.status !== 'all' && [externalFilter.status]

Comment (romanortynskyi) [reply src/components/enhanced-table/EnhancedTable.jsx]: It can. I fixed it.

**Review by Tolik170 — APPROVED** [review]

**Review by Yurenko — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**